src/store/activity/actions.js

```diff
@@ -276,3 +350,9 @@ export function resetActivities({ isClearingStorage = true } = {}) {
     type: ActionTypes.ACTIVITIES_RESET,
   };
 }
+
+export function resetMutualActivities() {
```

Comment:
Where is this used? and why is it needed?

Reply:
corrected